From e79e6cc1600d0f9607d709e3d047ba10fc3119ba Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Mon, 31 Oct 2022 08:28:57 +0800
Subject: [PATCH] rdk/grr/deliver11/ies-api: Add -Wno-error=address to be
 compiled with GCC12

Remove error: the comparison will always evaluate as 'true' for the
address of 'action' will never be NULL [-Werror=address]

Upstream-Status: Inappropriate [RDK is released in tarball form only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 user_modules/ies-api/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/user_modules/ies-api/meson.build b/user_modules/ies-api/meson.build
index 82469b5..177b2f7 100644
--- a/user_modules/ies-api/meson.build
+++ b/user_modules/ies-api/meson.build
@@ -16,6 +16,7 @@ default_options : ['c_std=c99', 'cpp_std=c++17', 'libdir=lib', 'buildtype=debugo
 meson_version : '>=0.55.0'
 )
 
+add_global_arguments('-Wno-error=address', language : 'c')
 
 ####################### API BUILD ############################
 
-- 
2.34.1

